### PR TITLE
Release 2.1.0

### PR DESCRIPTION
**Pull request by DarkGigaByte**

### CODE REVIEW

**Comment by bastelfreak** [comment]
can you please squash this? also this should be merged after https://github.com/voxpupuli/puppet-gluster/pull/61

**Comment by bastelfreak** [comment]
HI @DarkGigaByte, please put the snippet from [here](https://github.com/travis-ci/dpl/issues/487#issuecomment-236660825) into the .travis.yml. Otherwise I'm not able to release. Also please squash the above commits into one.

File: .travis.yml [hunk 1]
```diff
@@ -35,6 +35,8 @@ notifications:
   email: false
 deploy:
   provider: puppetforge
+  edge:
+    branch: ha-bug-puppet-forge
```
Comment (igalic) [inline .travis.yml]: huh??

Comment (bastelfreak) [inline .travis.yml]: this is needed to deploy to the forge until https://github.com/travis-ci/dpl/issues/487 is merged

**Pull request merged**